docs/MicrosoftAuth.md

```
@@ -2,14 +2,14 @@
 
 Authenticating with Microsoft is fully supported by Helios Launcher.
 
-## Acquiring an Azure Client ID
+## Acquiring an Entra Client ID
 
 1. Navigate to https://portal.azure.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)